service/worker/deployment/deployment_activities.go

```diff
@@ -44,7 +44,8 @@ type (
 	}
 
 	DeploymentNameWorkflowActivityInput struct {
-		DeploymentName string
+		deploymentName string
```

Comment:
this is an activity input, right? so it should be plain data (no behavior) and all fields must be exported so it can be serialized